src/cpp/desktop/DesktopMain.cpp

```diff
@@ -265,6 +265,60 @@ bool useRemoteDevtoolsDebugging()
 #endif
 }
 
+void initializeRenderingEngine(std::vector<char*>* pArguments)
+{
+   QString engine = desktop::options().desktopRenderingEngine();
```

Comment:
If I'm not mistaken then this means that users who are somehow totally unable to launch RStudio could get unblocked by seeding an options file (which is great!)